website/content/docs/agent-and-proxy/agent/process-supervisor.mdx

```diff
@@ -0,0 +1,47 @@
+---
+layout: docs
+page_title: Vault Agent's Process Supervisor Mode
+description: >-
+  Vault Agent's Exec functionality allows Vault secrets to be injected into a
+  process via environment variables using Consul Template markup.
+---
+
+# Vault Agent Exec
+
+Vault Agent's Exec functionality allows Vault secrets to be injected into a
+process via environment variables using [Consul Template markup][consul-templating-language].
+
+<Tip title="Kubernetes">
+  If you are running your applications in a Kubernetes cluster, we recommend evaluating the
```

Comment:
Should we also recommend the Vault Agent Injector here? I'm a little confused why this recommendation is on this page, and why we're only suggesting one of our approaches

Reply:
I know this PR is merged now, but I'm still interested in a response to this if you have it!

I'm a little confused why we have this recommendation on this page, and why we're only suggesting one approach.

Reply:
that's my bad, meant to update it yesterday after your note but forgot about it! will add it

Reply:
Is there a reason why you think the agent supervisor mode would be inappropriate for Kubernetes, though? Just for my own benefit, more than anything, since I can't see anything mysefl